woocommerce/woocommerce-android
# Show red budge on the more button till More screen is visible at least once

Closes: #7019


### Description

The PR makes the red badge invisible after a user see the "more" screen at least once

### Testing instructions

- Open the app and notice that there is a red badge on the "more" button
- Click on the "more" button and notice that the red badge is gone

### Images/gif


https://user-images.githubusercontent.com/4923871/180968897-54169497-103a-4320-9c6d-cf2a24701d69.mp4
